language/move-prover/src/cli.rs

```diff
@@ -143,6 +146,11 @@ impl Options {
                     .long("generate-only")
                     .help("only generate boogie file but do not call boogie"),
             )
+            .arg(
+                Arg::with_name("only-verify-spec")
+                    .long("only-verify-spec")
+                    .help("whether to only verify the functions having their specification defined"),
```

Comment:
Perhaps change to "whether to only verify functions which have associated specifications"

Reply:
Changed.